components/concept/storage/src/main/java/mozilla/components/concept/storage/HistoryStorage.kt

```diff
@@ -145,6 +154,17 @@ enum class RedirectSource {
 
 data class PageObservation(val title: String?)
 
+/**
+ * Information about a top frecent site.
```

Comment:
I'd expand this a tiny bit explain what "frecent" is.

Reply:
Fixed.